src/client/components/ProductsList/ProductsList.js

```diff
@@ -16,13 +17,26 @@ class ProductsList extends Component {
     this.setState(({ limit }) => ({ limit: limit + DISPLAY_LIMIT }));
   };
 
-  toggleItem = (id, isOrdered) => {
+  toggleItem = (author, id, isOrdered) => {
     const { toggle } = this.props;
-    toggle(id, isOrdered);
+    const {
+      currentUser: { name }
+    } = this.props;
+
+    /**
+     * If an item was ordered, check item author
+     * and update author if neccessary to point
+     * who is ordering this time.
+     */
+    !isOrdered
```

Comment:
please don't use ternary operator nested. This comment should not be needed - code should be readable. Please use variables with meaningful names.

Reply:
I have uploaded two versions of this piece of code. 
Check it out and drop me a line if any is acceptable